src/ir_Hitachi.cpp

```diff
@@ -1704,3 +1706,174 @@ String IRHitachiAc264::toString(void) const {
+void IRHitachiAc296::setPower(const bool on) {
+  _.Power = on;
+}
+
+/// Change the power setting to On.
+void IRHitachiAc296::on(void) { setPower(true); }
+
+/// Change the power setting to Off.
+void IRHitachiAc296::off(void) { setPower(false); }
+
+/// Get the operating mode setting of the A/C.
+/// @return The current operating mode setting.
+uint8_t IRHitachiAc296::getMode(void) const {
+  return _.Mode;
+}
+
+/// Set the operating mode of the A/C.
+/// @param[in] mode The desired operating mode.
+void IRHitachiAc296::setMode(const uint8_t mode) {
+  uint8_t newMode = mode;
+  switch (mode) {
+    case kHitachiAc296Heat:
+    case kHitachiAc296Cool:
+    case kHitachiAc296Auto: break;
+    default: newMode = kHitachiAc296Auto;
+  }
+
+  _.Mode = newMode;
+}
+
+/// Get the current temperature setting.
+/// @return The current setting for temp. in degrees celsius.
+uint8_t IRHitachiAc296::getTemp(void) const {
+  return _.Temp;
+}
+
+/// Set the temperature.
+/// @param[in] celsius The temperature in degrees celsius.
+void IRHitachiAc296::setTemp(const uint8_t celsius) {
+  uint8_t temp;
+  temp = std::min(celsius, kHitachiAc296MaxTemp);
+  temp = std::max(temp, kHitachiAc296MinTemp);
+  _.Temp = temp;
+}
+
+/// Get the current fan speed setting.
+/// @return The current fan speed.
+uint8_t IRHitachiAc296::getFan(void) const {
+  return _.Fan;
+}
+
+/// Set the speed of the fan.
+/// @param[in] speed The desired setting.
+void IRHitachiAc296::setFan(const uint8_t speed) {
+  uint8_t newSpeed = speed;
+  newSpeed = std::max(newSpeed, kHitachiAc296FanSilent);
+  newSpeed = std::min(newSpeed, kHitachiAc296FanAuto);
+
+  _.Fan = newSpeed;
+}
+
+/// Get a PTR to the internal state/code for this protocol.
+/// @return PTR to a code for this protocol based on the current internal state.
+uint8_t *IRHitachiAc296::getRaw(void) {
+  setInvertedStates();
+  return _.raw;
+}
+
+/// Set the internal state from a valid code for this protocol.
+/// @param[in] new_code A valid code for this protocol.
+/// @param[in] length Size (in bytes) of the code for this protocol.
+void IRHitachiAc296::setRaw(const uint8_t new_code[], const uint16_t length) {
+  memcpy(_.raw, new_code, std::min(length, kHitachiAc296StateLength));
+}
+
+#if DECODE_HITACHI_AC296
+bool IRrecv::decodeHitachiAc296(decode_results *results, uint16_t offset,
+                                const uint16_t nbits,
+                                const bool strict) {
+  uint16_t used = matchGeneric(results->rawbuf + offset, results->state,
+                               results->rawlen - offset, nbits,
+                               kHitachiAcHdrMark, kHitachiAcHdrSpace,
+                               kHitachiAcBitMark, kHitachiAcOneSpace,
+                               kHitachiAcBitMark, kHitachiAcZeroSpace,
+                               kHitachiAcBitMark, kHitachiAcMinGap, true,
+                               kUseDefTol, 0, false);
+  if (used == 0) return false;
+
```

Comment:
You should add a check for inverted byte pairs here (when `strict` is set,.) to confirm it's a valid message.